### PR TITLE
add browse function, update to supported APIs, remove default key mappings

### PR DESCRIPTION
**Pull request by drewrothstein**
-Added browse functionality
-Removed default key suggestions since > 4 on initialize creates a problem
-Updated API usage to use `query()` instead of `getSelected()`

Background: I was looking for a plugin to set custom key shortcuts for moving between tabs on *nix since the defaults were not comfortable to me.  This now lets me set the browsing between tabs to Ctrl+Right Arrow and Ctrl+Left Arrow.

Tested: Fedora 25, 4.8.15-300, Chrome: 55.0.2883.87

Happy to update the Screenshot if you would like.

### CODE REVIEW

File: src/manifest.json
Comment (danhp) [inline src/manifest.json]: Don't change the version number as maintainers should decide how to this is incremented.

File: src/background.js
Comment (danhp) [inline src/background.js]: Only search the `lastFocusedWindow`

**Review by danhp — CHANGES_REQUESTED** [review]

File: src/background.js
Comment (danhp) [inline src/background.js]: Add `lastFocusedWindow: true` or else it won't work with multiple windows.

File: src/background.js
Comment (danhp) [inline src/background.js]: `lastFocusedWindow: true`

File: src/background.js
Comment (danhp) [inline src/background.js]: `lastFocusedWindow: true`

File: src/background.js
Comment (danhp) [inline src/background.js]: `lastFocusedWindow: true`

**Review by danhp — CHANGES_REQUESTED** [review]

File: src/background.js
Comment (drewrothstein) [reply src/background.js]: Good call, updated.

File: src/background.js
Comment (drewrothstein) [reply src/background.js]: Updated.

File: src/manifest.json
Comment (drewrothstein) [reply src/manifest.json]: Sure, some maintainers like the bump too, reverted.

File: src/background.js
Comment (drewrothstein) [reply src/background.js]: Updated.

File: src/background.js
Comment (drewrothstein) [reply src/background.js]: Updated.

File: src/background.js
Comment (drewrothstein) [reply src/background.js]: Updated.

**Review by danhp — APPROVED** [review]

**Comment by danhp** [comment]
Thanks for this @drewrothstein 
Turns out I needed this too

**Comment by drewrothstein** [comment]
:)

**Comment by danhp** [comment]
Just posted the new version to the store, it should be updated soon.

**Comment by drewrothstein** [comment]
Spectacular.  Would you like a PR w/an updated screengrab?  I am happy to prep. one that shows the new options.

**Comment by danhp** [comment]
That would be great :)

**Pull request merged**
